USAGE_android.md

```diff
@@ -863,6 +863,15 @@ optional arguments:
                         Swap the swapchain color space if unsupported by replay device.
                         Check if color space is not supported by replay device and swap
                         to VK_COLOR_SPACE_SRGB_NONLINEAR_KHR. (forwarded to replay tool).
+  --offscreen-swapchain-frame-boundary
+                        Should only be used with offscreen swapchain. Activate
+                        the extension VK_EXT_frame_boundary (always supported
+                        if trimming, check for driver support otherwise) and
+                        insert command buffer submission with
+                        VkFrameBoundaryEXT where vkQueuePresentKHR was called
+                        in the original capture. This allows to preserve frames
+                        when capturing a replay that uses. offscreen swapchain.
+                        (forwarded to replay tool)
```

Comment:
```suggestion
                        Should only be used with offscreen swapchain. Activates
                        the extension VK_EXT_frame_boundary (always supported
                        if trimming, checks for driver support otherwise) and
                        inserts command buffer submission with
                        VkFrameBoundaryEXT where vkQueuePresentKHR was called
                        in the original capture. This allows preserving frames
                        when capturing a replay that uses offscreen swapchain.
                        (forwarded to replay tool)
```